securedrop_client/gui/widgets.py

```diff
@@ -1963,16 +1999,26 @@ def __init__(
         file_missing.connect(self._on_file_missing, type=Qt.QueuedConnection)
 
     def eventFilter(self, obj, event):
-        if event.type() == QEvent.MouseButtonPress:
+        t = event.type()
+        if t == QEvent.MouseButtonPress:
             if event.button() == Qt.LeftButton:
                 self._on_left_click()
+        # HERE BE DRAGONS. Usually I'd wrap this in an if not self.download,
```

Comment:
interesting....

Reply:
I think you can remove this comment along with the `HoverMove` check. This does not crash for me but your comment is somewhat vague. I think it would be more useful to developers in the long run if you create an issue instead of leaving in this comment.

Reply:
this is also a nit

Reply:
Hmm... so I needed `HoverMove` on my Ubuntu machine for this to work for me.

I'd *really* welcome the opportunity to de-brief about this crash. Honestly, it was a completely frustrating and the comment could certainly be re-written to make it clearer why the code is written in the way it is (the comment was written after a long and demoralising bug hunt - so probably sounds rather frazzled).

I also think it's best to keep such a (re-written) comment in the context of the code rather than a GitHub issue that folks forget and perhaps eventually close. I don't know whether this is a UK thing, but I've seen (and benefited from) such comments in several previous projects. Happy to be guided by you though..!